data/card_catalogue/upgradeable-1.json

```diff
@@ -187,7 +174,7 @@
     "tier": "red",
     "details": {
       "rules": {
-        "active": "Hover up to 2 metres off the ground until you lose concentration.",
+        "active": "Hover up to 20 metres off the ground until you lose concentration.",
```

Comment:
At red this deserved to be way stronger. With 20m I think you can bypass a lot of stuff, so I think it's good enough.